core/src/main/scala/cats/data/NonEmptyCollection.scala

```diff
@@ -40,6 +40,7 @@ private[cats] trait NonEmptyCollection[+A, U[+_], NE[+_]] extends Any {
   def filter(f: A => Boolean): U[A]
   def filterNot(p: A => Boolean): U[A]
   def collect[B](pf: PartialFunction[A, B]): U[B]
+  def collectFirst[B](pf: PartialFunction[A, B]): Option[B]
```

Comment:
did we add new tests for this? I don't see it.

Reply:
Yep you're right, I missed this one - added in 5c15c530a69d2c718ffdcb9850f88f7e1789445d